PWGEM/PhotonMeson/TableProducer/skimmerGammaConversions.cxx

```diff
@@ -128,10 +214,14 @@ struct skimmerGammaConversions {
   void processMc(aod::McCollision const& theMcCollision,
                  soa::SmallGroups<soa::Join<aod::McCollisionLabels,
                                             aod::Collisions>> const& theCollisions,
+                 aod::BCsWithTimestamps const& bcs,
```

Comment:
This is how it is done in the tutorials (https://github.com/AliceO2Group/O2Physics/blob/master/Tutorials/src/efficiencyPerRun.cxx) and other tasks. It should be right but there is never a guarantee. I added a couple of thinks to make it "Train save".

Reply:
I mean in this tutorial this is not completely the same because there the first argument of the process function is aod::Collision, not aod::McCollision. So I would probably ask in the O2Analysis channel to be on the safe side

Reply:
So as far as i understand their answer the BC table in not grouped with the other stuff. But in my opinions that should be fine since the magentic field will not change for the same run number so we can just use the first one and it should be fine. I think in the lambdakzero builder the BC is somehow mapped to the Collision with (auto bc = collision.bc_as<aod::BCsWithTimestamps>();) but this is only a guess because i cannot find any documentation on this piece of code. Nevertheless i think it should be fine to just use the first one

Reply:
this was just confirmed.

Reply:
great, thanks for clearing this!